pkg/cmd/run/rerun/rerun.go

```diff
@@ -0,0 +1,121 @@
+package rerun
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/cli/cli/api"
+	"github.com/cli/cli/internal/ghrepo"
+	"github.com/cli/cli/pkg/cmd/run/shared"
+	"github.com/cli/cli/pkg/cmdutil"
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type RerunOptions struct {
+	HttpClient func() (*http.Client, error)
+	IO         *iostreams.IOStreams
+	BaseRepo   func() (ghrepo.Interface, error)
+
+	RunID string
+
+	Prompt bool
+}
+
+func NewCmdRerun(f *cmdutil.Factory, runF func(*RerunOptions) error) *cobra.Command {
+	opts := &RerunOptions{
+		IO:         f.IOStreams,
+		HttpClient: f.HttpClient,
+	}
+
+	cmd := &cobra.Command{
+		Use:    "rerun [<run-id>]",
+		Short:  "Rerun a given run",
+		Args:   cobra.MaximumNArgs(1),
+		Hidden: true,
```

Comment:
Do we need to hide subcommands if the parent command is already hidden?

Reply:
i don't think so, it was just a reflex.